tests/helpers/order-sample.js

```diff
@@ -0,0 +1,55 @@
+export default function () {
```

Comment:
The main change is to change to a function, so everytime it's called a new object is generated.

This prevents me from manually cloning the object and modify it without affecting other copy of the object